src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -433,6 +433,22 @@ private void UpdateMSiTableGrid(Database msidb, string tableName)
 					// NOTE: Deliberately not calling msidb.TableExists here as some System tables could not be read due to using it.
 					string query = string.Concat("SELECT * FROM `", tableName, "`");
 
+					bool hasSequence = false;
+					using (var view = new ViewWrapper(msidb.OpenExecuteView(query)))
+					{
+						foreach (ColumnInfo col in view.Columns)

```

Comment:
Not an issue, more of a nit, but you can avoid the verbosity of a for foreach with [C#'s linq syntax](http://csharp.academy/linq-where/) in instances like this. For example, I think the equivalent of your foreach block here would be `hasSequence = view.Columns.Where((col) => col.Name == "Sequence").Count() > 0;`